getting-started/writing-and-serving-policy/policy-style.markdown

```diff
@@ -331,6 +331,38 @@ Which one do you prefer?
 
 ## Naming Conventions
 
+Naming conventions can also help to provide clarity.
+
+### Snakecase
+
+Words delimited by an underscore. This style is prevalant for *variables*,
+*classes*, *bundle* and *body* names in the Masterfiles Policy Framework.
+
+
+[%CFEngine_include_example(style_snake_case.cf)%]
+
+### Pascalecase
+
+Words delimited by capital Letters.
```

Comment:
Should mention where/when we encourage using this.

Reply:
I don't think we do suggest the use of this. I added it here because it's good consideration for someone in developing their own style guide.